### ISSUE TITLE
"Sign in with Matrix ID" doesn't look like an option

### ISSUE THREAD

**Issue by Mikaela** [vector-im/riotX-android#1731]
This may be related to #1730, I didn't understand that I can touch the "Sign in with Matrix ID" and thus went through the third option as I know the details I have to enter. I think this could be improved by it having a similar box as the other options.

Relatedly are new users who don't want to pay for Element Matrix Services or join millions on overloaded Matrix.org (which is a bit infamous in rooms I am in) expected to know to press "Customized and *complicated* settings"?

![screenshot from yesterday](https://matrix.feneas.org/_matrix/media/r0/download/matrix.feneas.org/b806f96191f6aa294d865755179d0acae3f1af42)

* Settings --> Accessibility --> high contrast text (under experimental)
* Nokia 1 / TA-1047 / Android 10 Go Edition

**Comment by olmari** [comment]
Also, shouldn't "sing in with matrix id" be pretty much first option within signing in? :)

**Comment by frandavid100** [comment]
I agree. If we want matrix.org to stop being massive and overloaded, we should stop showcasing it as a first option.
